src/test/java/spoon/test/template/testclasses/InvocationTemplate.java

```diff
@@ -15,15 +15,15 @@ void invoke() {
 	
 	@Local
 	public InvocationTemplate(CtTypeReference<?> ifaceType, String methodName) {
-		this.ifaceType = ifaceType;
-		this.methodName = methodName;
+		this._IFace = ifaceType.getSimpleName();

```

Comment:
The result must be `a.b.c.Foo`. Note that IFace is (or should be) marked @Local and is here to represent arbitrary interface of the java world.